crates/sui-graphql-e2e-tests/tests/epoch/chain_identifier.exp

```diff
@@ -0,0 +1,14 @@
+processed 3 tasks
+
+init:
+C: object(0,0)
+
+task 1 'create-checkpoint'. lines 8-8:
+Checkpoint created: 1
+
+task 2 'run-graphql'. lines 10-13:
+Response: {
+  "data": {
+    "chainIdentifier": "8f58ad28"
```

Comment:
Will not this be different every time?

Reply:
Tansactional test runner makes sure that this isn't the case.